meteor-app/imports/ui/assessment/assessment-service-card-list.js

```diff
@@ -38,7 +38,7 @@ class ServiceList extends Component {
                     />
                 </Grid.Column>
                 <Grid.Column width={6}>
-                    <Button size='big' fluid secondary onClick={() => this.props.onClick()}>Custom Service</Button>
+                    <Button size='big' fluid secondary onClick={this.props.onClick}>Custom Service</Button>
```

Comment:
instead of naming a function `onClick`, which doesn't give me any clue as to what it does, give it a meaningful name, such as `selectCustomService`, which will improve the readability of this code immensely